services/QuillLMS/app/services/classlink_integration/instructor_linker.rb

```diff
@@ -39,6 +42,13 @@ def run
       )
     end
 
+    private def auto_verify_teacher
+      return if persisted_user.teacher_verification_status == TeacherInfo::APPROVED
```

Comment:
Clever has a disjunct "teacher.admin_approval_status == AdminInfo::APPROVED".  Does this not apply to Classlink?  Maybe a question for @happythenewsad

Reply:
@brendanshean In the ClassLink flow, we are already checking to see if the user is an admin before running this branching logic, and the admin path is going to give them an approved `AdminInfo` record instead. Here, the existing logic doesn't handle admins (on the Quill side), so I just didn't want to create extra records if it's not necessary. Similarly, this code won't run here if the user isn't associated with a school, but that's not true for the Clever path, so I'm checking that there as well.